src/styles/components/_nav.scss

```diff
@@ -385,4 +385,418 @@
     color: $azure;
     font-weight: 600;
   }
+}
+
+#main-header-2022 {
```

Comment:
Note that old header and footer styles are still included in these files, so the pages which have not yet applied the new header and footer (via main-giraffe) will not look completely broken.